classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label, :students
```

Comment:
- [ ] There is no need to change the `student` instance variable from outside the Classroom. therefor please use the attribute reader for this variable.